examples/knowledgebase/query.py

```
@@ -0,0 +1,185 @@
+
+    console_handler = logging.StreamHandler(sys.stdout)
+    console_handler.setLevel(logging.INFO)
+
+    formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    console_handler.setFormatter(formatter)
+
+    logger.addHandler(console_handler)
+    return logger
+
+
+async def query_knowledge_base(query: str, limit: int = 5, center_node_uuid: Optional[str] = None) -> None:
```

Ruff (0.8.2)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)